src/concept/answer/ConceptMapGroup.jl

```diff
@@ -0,0 +1,18 @@
+# This file is a part of TypeDBClient.  License is MIT: https://github.com/Humans-of-Julia/TypeDBClient.jl/blob/main/LICENSE
+
+struct ConceptMapGroup
+    owner::AbstractConcept
+    concept_maps::Vector{ConceptMap}
+end
```

Comment:
https://docs.julialang.org/en/v1/manual/performance-tips/#man-performance-abstract-container

Try to use concrete types in structs. Perhaps make it parametric?
```suggestion
struct ConceptMapGroup{T <: AbstractConcept}
    owner::T
    concept_maps::Vector{ConceptMap}
end
```